includes/parameters.h

```diff
@@ -25,7 +25,7 @@
 ///@{
 
 // Période de traitement en millisecondes
-#define PCONTROLLER_COMPUTE_PERIOD 10u
+#define PCONTROLLER_COMPUTE_PERIOD_US 10000u
```

Comment:
We should update the previous comment accordingly to this modification.

Reply:
Done !